src/serverenvironment.cpp

```diff
@@ -1219,6 +1219,11 @@ void ServerEnvironment::step(float dtime)
 		}
 	}
 
+	if (m_database_check_interval.step(dtime, 5.0f)) {
```

Comment:
I know this is nitpicking but wouldn't 10 work just as well?

Reply:
i think 10 can suffice yes, but more longer can lead to timeouts in some configurations